projects/angular/src/layout/breadcrumbs/_breadcrumbs.clarity.scss

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+@use 'sass:meta';
+@use '../../utils/mixins';
+@use 'variables.breadcrumbs' as breadcrumb-variables;
+@use '@cds/core/tokens/tokens.scss';
+@include meta.load-css('properties.breadcrumbs');
+
+.clr-breadcrumb-menu {
```

Comment:
This file should be wrapped in `@include mixins.exports('breadcrumbs.clarity') {}`